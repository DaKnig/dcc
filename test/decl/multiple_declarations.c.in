int x;
float y=9, z=10;
