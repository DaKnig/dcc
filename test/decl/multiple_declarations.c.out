tree representation of multiple_declarations.c.in:
x is a 
i16 ;
y is a 
f32 =
|   -> (int)9
;
z is a 
f32 =
|   -> (int)10
;
